docs/OMS-MySQL-Instructions.md

```diff
@@ -0,0 +1,88 @@
+# MySQL Server and Log Monitoring Solution for Operations Management Suite
+
+1. Setup a Linux(Ubuntu/Redhat) machine and install [MySQL](http://dev.mysql.com/doc/refman/5.7/en/installing.html).
```

Comment:
Change to "Setup a Linux server"

We support additional OSes besides Ubuntu and Red Hat